### PR TITLE
[UI] Navigation Menu Icon Scale with zoom in/out 

### LINKED ISSUE #1712: [UI] Navigation Menu Icon Not Scaling
**Description**
The lefthand navigation menu icons do not scale on browser zoom.

**Expected Behavior**
The icons should enlarge/shrink as the user zooms their view.

**Screenshots**
Screen Shot 2020-10-20 at 9 01 00 PM

### PR DESCRIPTION
**Pull request by Ayush-Khandelwal-007**
Signed-off-by: Ayush ayushkhandelwal3217@gmail.com

This PR fixes #1712 
This issue was browser-specific.
I guess now it will be working fine in all browsers.

### CODE REVIEW

File: ui/components/Navigator.js [hunk 1]
```diff
@@ -235,15 +240,16 @@ const categories = [
   },
   {
     id: "Performance",
-    icon: <FontAwesomeIcon icon={faTachometerAlt} transform="shrink-2" fixedWidth />,
+    icon:
+      <FontAwesomeIcon icon={faTachometerAlt} transform="shrink-2" style={{ height: "1.21rem", width: "1.21rem", fontSize: "1.21rem" }} />,
```
Comment (Tanuj22) [inline ui/components/Navigator.js:244]: @Ayush-Khandelwal-007 can't the `drawerIcons` class be used here instead of adding style to each of the icons?

**Review by Tanuj22 — COMMENTED** [review]
I believe this was a safari related issue so I can't test it out. @leecalcote I guess you will have to check it out :sweat_smile:

File: ui/components/Navigator.js (hunk 1 above)
Comment (Ayush-Khandelwal-007) [reply ui/components/Navigator.js:244]: okay , I'll just check it and update to the cleanest way possible.

Comment (Ayush-Khandelwal-007) [reply ui/components/Navigator.js]: I can't use class `drawerIcons ` for the elements defined outside the class. So I just cleaned it up a bit.
and can you please help me resolve the issue in the below checks?

**Review by Tanuj22 — APPROVED** [review]
LGTM! The ci failure is not due to your changes ig

**Comment by Ayush-Khandelwal-007** [comment]
yes, I didn't made any changes in any go files but still this is failing. Is this a point of concern?

**Comment by leecalcote** [comment]
@Ayush-Khandelwal-007 not anymore. We're good to go.

**Comment by leecalcote** [comment]
Thank you for this review, @Tanuj22!

**Comment by leecalcote** [comment]
Good work, @Ayush-Khandelwal-007 👍

**Pull request merged**